matter/wifi/rs911x/hal/efx_spi_intf.c

```diff
@@ -186,13 +186,13 @@ void sl_wfx_host_reset_chip(void)
   GPIO_PinOutClear(WFX_RESET_PIN.port, WFX_RESET_PIN.pin);
 
   // Delay for 10ms
-  vTaskDelay(pdMS_TO_TICKS(DELAY_10MS));
```

Comment:
Why the revert ?

Reply:
I find the revert ok for task delays like this to be honest it is clear enough

Reply:
I have kept the revert.